examples/ts_openai_client/05-imperative-fetch-based.ts

```
@@ -0,0 +1,44 @@
+import { FunctionCall, humanlayer } from "humanlayer";
+
+const hl = humanlayer({ verbose: true, runId: "ts-imperative-fetch-based" });
+
+const demoEscalation = async () => {
+  const call: FunctionCall = await hl.createFunctionCall({
+    spec: {
+      fn: "multiply",
+      kwargs: { foo: "bar" },
+      channel: {
+        email: {
+          address:
+            process.env.HL_EXAMPLE_CONTACT_EMAIL || "dexter@humanlayer.dev",
+        },
+      },
+    },
+  });
+
+  const escalated_call: FunctionCall = await hl.escalateEmailFunctionCall(
+    call.call_id,
+    {
+      escalation_msg: "please take a look because it's been too long",
+      additional_recipients: [
+        {
+          address: process.env.HL_EXAMPLE_SECOND_CONTACT_EMAIL || "",
```

should we have a default here? or fail if its unset?